Makefile

```diff
@@ -0,0 +1,4 @@
+test:
```

Comment:
Is there a good reason for wrapping this with make?  Running `pytest` directly would seem fine, or maybe using `setuptools` as a first step towards turning this into a proper Python package?

Reply:
For convenience. It's not really a python project, so I don't think it's a great idea to just drop in a setup.py file to make pytest happy.

Reply:
I actually think we should eventually have a kernelci python package, but that wouldn't include the jenkins stuff and we're not quite there yet.  So I agree that using a `setup.py` wouldn't be very appropriate at the moment.  Still I'm not convinced either by the `Makefile` approach just to run `pytest`...

How about adding something about `pytest` in the README instead?

Reply:
That's the point though - you can't just run 'pytest'. You must run `python -m pytest`.

I think it's a larger question than just pytest. What if we want to enforce for example, flake8, or black, along with pytest? There should be a simple, obvious one-liner that any developer can run to guarantee compliance and match whatever automated CI will also run against pull requests. Make provides a nice interface for that automatically. We can also write some shell script, or rely on documentation, but I don't think it's a good developer experience to ask them to copy paste multiple commands out of documentation.

Once we decide on the interface though, I agree it needs to be added to README and I can do that in this PR or a subsequent PR.

Reply:
OK, fair enough.  If it happens to make more sense to have these test commands elsewhere than in the `Makefile` in the future, it won't be hard to make the change then.